firmware/user/display/oled.h

```diff
@@ -35,6 +35,7 @@ bool initOLED(bool reset);
 void drawPixel(int16_t x, int16_t y, color c);
 color getPixel(int16_t x, int16_t y);
 bool ICACHE_FLASH_ATTR setOLEDparams(bool turnOnOff);
+int ICACHE_FLASH_ATTR updateOLEDScreenRange( uint8_t minX, uint8_t maxX, uint8_t minPage, uint8_t maxPage );
```

Comment:
`updateOLED()` is already called on a timer. If you're exposing `updateOLEDScreenRange()` are you planning on a way to turn off the periodic `updateOLED()` calls?

Reply:
I was not planning on it - should I?  I just figured I'd let the rest of the system do its thing.  this just explicitly schleps the pixels to the display.

Reply:
No, the behavior is fine as-is. I saw the declaration in a header, so I thought you were planning on using it outside `oled.c`. It's only used in there now, so the declaration could be in `oled.c` to keep scope a little tighter.

Reply:
I think it may make sense to expose to folks.  Like the games know what part of the screen they're updating.